src/web/web-service/ot_client.cpp

```diff
@@ -118,6 +118,25 @@ char *OpenThreadClient::Execute(const char *aFormat, ...)
     ssize_t count;
     size_t  rxLength = 0;
 
+    // clear any buffered output
+    for (;;)
+    {
+        fd_set  readFdSet;
+        timeval timeout = {0, 0};
+
+        FD_ZERO(&readFdSet);
+        FD_SET(mSocket, &readFdSet);
+
+        ret = select(mSocket + 1, &readFdSet, nullptr, nullptr, &timeout);
+        if (ret <= 0)
+        {
+            break;
+        }
+
+        count = read(mSocket, mBuffer, sizeof(mBuffer));
+        VerifyOrExit(count > 0);
+    }
```

Comment:
nit: encapsulate with a function? `OpenThreadClient::Execute` seems to be getting large.